spec/features/admin_views_list_users_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+feature 'admin views a list of users' do
+
```

Comment:
I'd like a few more tests here: Can we add `scenario "admin sees list of all users"` that makes sure the users are displayed on the page?